Added "Health" property, changed "Potion" to "Thrown Potion"

T-727 opened the pull request:
- Added "Health" property, changed "Potion" to "Thrown Potion"
- Changed "Potion" to "Thrown Potion"

T-727 committed:
Added "Health" property, changed "Potion" to "Thrown potion"

- Added "Health" property, changed "Potion" to "Thrown potion"
- Changed "Potion" to "Thrown potion"

T-727 committed:
Capitalised "Not Applicable" lol

- Capitalised "Not Applicable"

11-90-an commented:
evoker fangs (898) and small fireball (966) are empty, Not Applicable for both maybe?

T-727 committed:
Fixed missing values

- Added "Not Applicable" to "Small Fireball"
- Added "Not Applicable" to "Evoker Fangs"
Thanks to 1190an

T-727 commented:
@11-90-an done
tyvm

JoakimThorsen commented on data/entity_data.json:
```diff
@@ -864,6 +864,142 @@
+		"health": {
+			"property_name": "Health": ,
+			"default_value": "Undefined": ,
+			"default_selection": true,
```
You have a few extra `:` characters here

JoakimThorsen commented on data/entity_data.json:
```diff
@@ -864,6 +864,142 @@
+		"Bat": 6,
+		"Bee": 10,
+		"Blaze": 20,
+		"Boat": ,
```
Missing Value

JoakimThorsen commented on data/entity_data.json:
```diff
@@ -864,6 +864,142 @@
+		"Piglin Brute": 50,
+		"Pillager": 24,
+		"Polar Bear": 30,
+		"Primed TNT": ,
```
Missing Value

JoakimThorsen commented on data/entity_data.json:
```diff
@@ -864,6 +864,142 @@
+		"Dragon Fireball": "Not Applicable",
+		"Drowned": 20,
+		"Elder Guardian": 80,
+		"End Crystal": ,
```
Missing Value

JoakimThorsen reviewed (changes requested):
There are still a few missing values and you have a few JSON formatting errors, including one too few closing curly braces. I'd recommend verifying your formatting by using something like https://jsonlint.com/

T-727 committed:
Fixed issues

- Missing values
- Formatting errors
Thanks to Joabro

JoakimThorsen commented:
That's better :P Thanks
